react-native-in-app-payments-quickstart/app/components/CardsOnFileTitleView.tsx

```diff
@@ -30,7 +30,7 @@ CardsOnFileTitleView.propTypes = {
   onCloseCardsOnFileScreen: PropTypes.func.isRequired,
 };
 
-export default function CardsOnFileTitleView({ onCloseCardsOnFileScreen }) {
+export default function CardsOnFileTitleView({ onCloseCardsOnFileScreen }: { onCloseCardsOnFileScreen: any }) {
```

Comment:
please use a concrete type instead of any

Reply:
Resolved it.